lib/open_food_network/payments_report.rb

```diff
@@ -100,7 +100,7 @@ def columns
          proc { |orders| orders.first.distributor.name },
          proc { |orders| orders.to_a.sum(&:item_total) },
          proc { |orders| orders.sum(&:ship_total) },
-         proc { |orders| orders.sum(&:outstanding_balance) },
```

Comment:
I think we were just missing https://www.rubydoc.info/stdlib/bigdecimal/BigDecimal#coerce-instance_method. I didn't know this was possible :heart_eyes: 

I wonder if we didn't spot this due to the Ruby upgrade that after this :thinking:

Reply:
It's really weird. I tried 5 or 6 times to write a test for this, in feature specs, controller specs, model specs... it passed without error every time! And we actually _do_ have a test for this already, and it was passing. Really easy to replicate in the browser or console though! :boom:

#WTF :man_shrugging: :man_shrugging: :man_shrugging:

Reply:
:thinking: :thinking: I was probably stubbing things I shouldn't?

Reply:
I would have expected the tests here to have failed:

https://github.com/openfoodfoundation/openfoodnetwork/blob/7d5726b6c799cd0229640eaa503d6474e64bae3d/spec/features/admin/reports/payments_report_spec.rb